octoprint_Spoolman/SpoolmanPlugin.py

```diff
@@ -49,6 +56,10 @@ def getSpoolmanConnector(self):
         )
 
     def triggerPluginEvent(self, eventType, eventPayload = {}):
+        # Füge das "plugin_Spoolman_" Präfix hinzu, wenn es noch nicht vorhanden ist
```

Comment:
Please translate all comments written in German to English to maintain consistency.